Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -3696,6 +3695,16 @@ internal static void OnPlayModeChange(PlayModeStateChange change)
             }
         }
 
+        // We have this function to hide away instanceId -> entityId migration that happened in Unity 6.4
+        public static bool HasNativeObject(Object obj)
```

Comment:
Minor: Hmmm.... This diff was hard to review but just a thought.

What if this code was changed so that `HasNativeObject(Object)` would call another function:

`Object GetObjectFromEntityID(EntityId id) { <same #if dance to map to EditorUtility> }`

And then using a conditional alias where needed for EntityId, e.g. 
```cs
#if !UNITY_6000_4_OR_NEWER
using EntityId = int;
#endif
```
Would that eliminate all other branches?